app/templates/_base_page.html

```diff
@@ -1,38 +1,62 @@
-{% extends "toolkit/layouts/_base_page.html" %}
+{% extends "govuk-frontend/template.njk" %}
 
 // Insert line for each component module import
 
-{% from "components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
-{% from "components/button/macro.njk" import govukButton %}
-{% from "components/error-summary/macro.njk" import govukErrorSummary %}
-{% from "components/input/macro.njk" import govukInput %}
+{% from "govuk-frontend/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk-frontend/components/button/macro.njk" import govukButton %}
+{% from "govuk-frontend/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk-frontend/components/input/macro.njk" import govukInput %}
+{% from "govuk-frontend/components/phase-banner/macro.njk" import govukPhaseBanner %}
 
 
-{# TODO: Remove once we start removing govuk_template, GOV.UK Frontend template uses pageTitle #}
-{% block page_title %}{% block pageTitle %}{% endblock %}{% endblock %}
+{# Import DM Components #}
+{% from "digitalmarketplace/components/header/macro.njk" import dmHeader%}
+{% from "digitalmarketplace/components/footer/macro.njk" import dmFooter%}
+
+{% set assetPath = '/user/static' %}
+
+{% block head %}
+  {% include "layouts/_custom_dimensions.html" %}
+  {% include "layouts/_site_verification.html" %}
+  <link type="text/css" rel="stylesheet" href="{{ asset_fingerprinter.get_url('stylesheets/application.css') }}" />
+  {% block pageStyles %}{% endblock%}
+{% endblock %}
+
+{% block header %}
+  {{ dmHeader({
+    "role": current_user.role | default(None)
+  }) }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    "tag": {
+      "text": "beta"
+    },
+    "html": 'Help us improve the Digital Marketplace - <a class="govuk-link" href="'  + url_for('external.help') + '">send your feedback</a>'
+  }) }}
+  {% block breadcrumb %}{% endblock%}
+  {% include "toolkit/flash_messages.html" %}
```

Comment:
Flash messages should be in `content` block so it doesn't get skipped by the skip link